src/components/Login/KakaoAuth.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { REST_API_KEY, REDIRECT_URI } from '../../components/OAuth';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import styled, { keyframes } from 'styled-components';
```

Comment:
```suggestion
import React, { useEffect } from 'react';
import { useNavigate } from 'react-router-dom';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faSpinner } from '@fortawesome/free-solid-svg-icons';
import styled, { keyframes } from 'styled-components';
import { REST_API_KEY, REDIRECT_URI } from '../../components/OAuth';
```

import 순서 수정해주세요! 일반적인 convention을 따라 순서만 잘 지켜주셔도 가독성이 좋아집니다. 
아래 순서 참고해주세요.

React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)